### ISSUE TITLE
File "conversion_factors.json" missing on compile

### ISSUE THREAD

**Issue by hl** [kipcole9/cldr_units#10]
When setting up a new project with: 

```elixir
defp deps do
  [
    {:jason, "~> 1.1"},
    {:ex_cldr_units, "~> 2.3"}
  ]
end
```

the compile steps fails with the following error:

```
==> ex_cldr_units
Compiling 8 files (.ex)

== Compilation error in file lib/cldr/conversion.ex ==
** (File.Error) could not read file "./priv/conversion_factors.json": no such file or directory
    (elixir) lib/file.ex:353: File.read!/1
    lib/cldr/conversion.ex:14: (module)
    (stdlib) erl_eval.erl:680: :erl_eval.do_apply/6
could not compile dependency :ex_cldr_units, "mix compile" failed. You can recompile this dependency with "mix deps.compile ex_cldr_units", update it with "mix deps.update ex_cldr_units" or clean it with "mix deps.clean ex_cldr_units"
```

I failed to see why the download mix task is not run and hope you can help.

**Comment by kipcole9** [comment]
@hl My apoiogies for the sloppy hex packaging which omitted the `./priv` directory.  Fixed now in [this commit](https://github.com/kipcole9/cldr_units/commit/3014190e9be2b48f2e372566a7114c4e9997bb41) and published on [hex as version 2.3.3](https://hex.pm/packages/ex_cldr_units/2.3.3).

Sorry for the inconvenience. If you:
```
mix deps.update ex_cldr_units
```
You should be good to go.

**Comment by hl** [comment]
@kipcole9 absolutely no apologies needed! I’m really appreciative of all your hard work. 

Thank you for taking the time to look at this issue.
